app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :author
+  belongs_to :posts
+
+  def likes_counter
+    post.update_column(:likes_counter, post.likes.count)
+  end
```

Comment:
- [ ] [Optional] It would be better to use increment! as increment! method doesn't want us to save it manually. why it is better you can see [here](https://stackoverflow.com/questions/11499110/increment-vs-1).
example only
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```